validator/client/src/main/java/tech/pegasys/teku/validator/client/KeyManager.java

```diff
@@ -92,8 +104,46 @@ public DeleteKeysResponse deleteValidators(final List<BLSPublicKey> validators)
    * @param slashingProtection a combined slashing protection payload
    * @return a list of 1 status per keystore that was attempted to be imported
    */
-  public List<ImportStatus> importValidators(
+  public List<PostKeyResult> importValidators(
       final List<String> keystores, final List<String> passwords, final String slashingProtection) {
-    throw new NotImplementedException("importValidators not implemented yet");
+
+    final List<PostKeyResult> postKeyResults = new ArrayList<>();
+
+    if (keystores.size() == passwords.size()) {
```

Comment:
It would be better to exit early here if the sizes don't match:
```
if (keystores.size() != passwords.size()) {
 return List.of(PostKeyResult.error("Keystores and passwords quantity must be the same."));
}
```

Although, I'm not sure that returning a single `PostKeyResult` error is the right thing if these don't match - it probably should throw `IllegalArgumentException` so that we respond with a 400 bad request.  Returning a single result suggests that the error is with the first keystore posted which is misleading.